### PR TITLE
FOPTS-1122 Update AWS Usage Reduction Policies (Unused Resources)

### PR DESCRIPTION
**Pull request by night-codes**
### Description

Update and standardize AWS Usage Reduction Policies:
- AWS Idle Compute Instances
- AWS Unused RDS Instance
- AWS Unused IP Addresses
- AWS Unused Volumes

### Issues Resolved

[FOPTS-1122 - AWS Usage Reduction Policies (Jira)](https://flexera.atlassian.net/browse/FOPTS-1122)

### Link to Example Applied Policy

[AWS Idle Compute Instances](https://app.flexera.com/orgs/1105/automation/applied-policies/projects/60073?policyId=649143454466b10001e21822)

[AWS Unused RDS Instance](https://app.flexera.com/orgs/1105/automation/applied-policies/projects/60073?policyId=64911f734466b10001e217ff)

[AWS Unused IP Addresses](https://app.flexera.com/orgs/1105/automation/applied-policies/projects/60073?policyId=6491077c4466b10001e217ea)

[AWS Unused Volumes](https://app.flexera.com/orgs/1105/automation/applied-policies/projects/60073?policyId=649102dd4466b10001e217e9)

### Contribution Check List

- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

**Review by XOmniverse — CHANGES_REQUESTED** [review]
Please hold off on PRs for AWS usage/rate reduction policies. We are in the process of completely revamping them, so the more changes that occur in the meantime, the more merge conflicts we are going to have to resolve.

File: cost/aws/idle_compute_instances/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v5.2
+
+- Fixed `tags` incident field output
+- Added `resourceName`, `lookBackPeriod`, `threshold`, `memoryThreshold` and `thresholdType` incident fields
```
Comment (XOmniverse) [inline cost/aws/idle_compute_instances/CHANGELOG.md:6]: The CHANGELOG should make reference to what the user would see when using the policy, not the code itself. These should be listed with the names the user would see when looking at an incident.

File: cost/aws/unused_ip_addresses/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 5.2
+
+- Added `resourceName` incident field
```
Comment (XOmniverse) [inline cost/aws/unused_ip_addresses/CHANGELOG.md:5]: Like the other CHANGELOG, this should refer to the incident fields by the name the user would see.

File: cost/aws/unused_rds/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 5.2
+
+- Added `databaseEngine`, `engineVersion`, `platform` and `lookBackPeriod` incident fields
```
Comment (XOmniverse) [inline cost/aws/unused_rds/CHANGELOG.md:5]: Like the other CHANGELOG, this should refer to the incident fields by the name the user would see.

File: cost/aws/unused_volumes/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v6.1
+
+- Added `resourceName`, `age` and `lookBackPeriod` incident fields
```
Comment (XOmniverse) [inline cost/aws/unused_volumes/CHANGELOG.md:5]: Like the other CHANGELOG, this should refer to the incident fields by the name the user would see.

**Review by XOmniverse — CHANGES_REQUESTED** [review]
Just some minor CHANGELOG changes requested. Otherwise, looks good.

File: cost/aws/unused_volumes/CHANGELOG.md (hunk 4 above)
Comment (night-codes) [reply cost/aws/unused_volumes/CHANGELOG.md]: Updated

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**